runtime/onert/core/src/util/ShapeInference.cc

```diff
@@ -27,8 +27,6 @@
 #include <sstream>
 #include <cmath>
 
-// TODO Move this file to runtime/onert/core/src/util
```

Comment:
@hyunsik-yoon And I just found this and removed it as it seems to be done already.